scipy/special/add_newdocs.py

```diff
@@ -4407,22 +4407,218 @@ def add_newdoc(place, name, doc):
     """
     kolmogi(p)
 
-    Inverse function to kolmogorov
+    Inverse Survival Function of Kolmogorov distribution
 
+    It is the inverse function to `kolmogorov`.
     Returns y such that ``kolmogorov(y) == p``.
+
+    Parameters
+    ----------
+    p : float array_like
+        Probability
+
+    Returns
+    -------
+    float
```

Comment:
Maybe write y : float and repeat that kolmogorov(y) = p here again for clarity? y is array_like as p I guess, same for the other functions below.